posthog/models/test/test_oauth.py

```diff
@@ -134,6 +144,68 @@ def test_can_create_application_with_http_redirect_url_when_debug_is_true(self):
             algorithm="RS256",
         )
 
+    valid_loopback_uris = [
```

Comment:
Since there's logic for it, it would be great to test that the `http://` protocol can only be used with localhost addresses and not with a public IP